src/typings.d.ts

```diff
@@ -63,3 +63,11 @@ declare namespace AV {
     stop();
   }
 }
+
```

Comment:
Paranoid that I was missing other errors that appeared when running tests, I declared these to get the Cannot find module errors to go away. However, then I just started seeing a Can't find name error instead. We're gonna need those typings.